lib/email.js

```diff
@@ -20,12 +24,20 @@ bedrock.events.on('bedrock.test.configure', function() {
   require('./test.config');
 });
 
-var logger = bedrock.loggers.get('app');
-
 var api = {};
 module.exports = api;
 
+bedrock.events.on('bedrock.start', function(callback) {
+  brNotifications.register(api, function(err, results) {
```

Comment:
> Do you mean something like bedrock-notifications firing off an "I'm ready" event that lets other modules subscribe to it?

Yeah.

> I like this way because other modules can potentially register or deregister themselves at different points in time (even thought we don't take advantage of that at the moment).

Ok. That's fine. It would be good to tell people a best practice (and the reasons for it), though.